src/kagglehub/config.py

```diff
@@ -127,10 +123,13 @@ def _is_env_var_truthy(env_var_name: str) -> bool:
 
 
 def set_kaggle_credentials(username: str, api_key: str):
+    if not username or not api_key:
```

Comment:
Can you check whether the username is empty or only whitespace (you can use trim for this)?

And add a test to ensure we throw a value error if the user enters " " (whitespace) to for either of these.